mkdocs.yml

```diff
@@ -16,6 +16,7 @@ pages:
   - 'MyOSG': 'SLA/myosg.md'
   - 'OASIS': 'SLA/oasis-replica.md'
   - 'OIM': 'SLA/oim.md'
+- External OASIS repositories: 'oasis.mid'
```

Comment:
I assume this should be 'oasis.md'.  But anyway, shouldn't it have 'external' in the file name in addition to oasis?